### ISSUE TITLE
Salt provisioner can't install Windows minions after version 2016.11.7

### ISSUE THREAD

**Issue by jonlabroad** [hashicorp/vagrant#8925]
When attempting to use a Salt minion version past 2016.11.7, the provisioner cannot find the installer when downloading:
```
Downloading Salt minion installer Salt-Minion-2017.7.1-AMD64-Setup.exe
powershell.exe : Exception calling "DownloadFile" with "2" argument(s): "The remote server retur
    + CategoryInfo          : NotSpecified: (Exception calli...te server retur:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
ned an error: (404) Not Found."
At C:\tmp\bootstrap_salt.ps1:56 char:24
+ $webclient.DownloadFile <<<< ($url, $file)
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : DotNetMethodException

Installing Salt minion...
C:\tmp\salt.exe /S /minion-name=stable
The term 'C:\tmp\salt.exe' is not recognized as the name of a cmdlet, function,
 script file, or operable program. Check the spelling of the name, or if a path
 was included, verify that the path is correct and try again.
At C:\tmp\bootstrap_salt.ps1:67 char:18
+   C:\tmp\salt.exe <<<<  /S /minion-name=$minion | Out-Null
    + CategoryInfo          : ObjectNotFound: (C:\tmp\salt.exe:String) [], Com
   mandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

This is because Salt has changed their installer filenames to include the Python version after 2016.11.7: https://repo.saltstack.com/windows/

**Comment by jonlabroad** [comment]
This is fixed by #8926

**Comment by jbonhag** [comment]
Closing this issue because it was reported as fixed. 🎉
